packages/amplication-server/src/core/resource/dto/BreakServiceToMicroserviceResult.ts

```diff
@@ -0,0 +1,58 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+import { EnumUserActionStatus } from "../../userAction/types";
+
+@ObjectType({
+  isAbstract: true,
+})
+class BreakTheMonolithDataModel {
+  @Field()
+  originalEntityId: string;
+
+  @Field()
+  name: string;
+}
+
+@ObjectType({
+  isAbstract: true,
+})
+class Microservice {
+  @Field()
+  name: string;
+
+  @Field()
+  functionality: string;
+
+  @Field(() => [BreakTheMonolithDataModel])
+  dataModels: BreakTheMonolithDataModel[];
+}
+
+@ObjectType({
+  isAbstract: true,
+})
+export class BreakTheMonolithRecommendationsResult {
+  @Field(() => [Microservice])
+  microservices: Microservice[];
+}
+
+@ObjectType({
+  isAbstract: true,
+})
+export class BreakServiceToMicroserviceResult {
+  @Field(() => EnumUserActionStatus, {
+    nullable: false,
+    description: "The status of the user action",
+  })
+  status: EnumUserActionStatus;
+
+  @Field(() => String, {
+    nullable: false,
+    description: "The original resource ID",
+  })
+  originalResourceId: string;
```

Comment:
```suggestion
  resourceId: string;
```

Reply:
I think `originalResourceId` provides more context and it's consistent with `originalEntityId` that we use in `BreakTheMonolithDataModel`